lib/templateFactory.js

```diff
@@ -258,12 +265,18 @@ class TemplateFactory extends BaseFactory {
 
             const templateIds = templates.map(t => t.id);
 
+            // all jobs with templates
             const metricJobsListConfig = {
                 params: { templateId: templateIds },
                 readOnly: true,
                 aggregationField: 'templateId'
             };
 
+            const metricPipelinesConfig = {
+                params: { templateId: templateIds },
+                readOnly: true
+            };
```

Comment:
This is not an optimal solution. This approach loads the all the fields of job objects from the DB to memory and perform aggregation here.

As we had discussed, we should enhance `datastore-sequelize` to accept input and generate below SQL query so that the aggregation is performed at source (DB)
```
select count(distinct(pipelineId)), templateId
from jobs
group by templateId;
```